### ISSUE TITLE
New user, odd behavior or am I misunderstand?

### ISSUE THREAD

**Issue by rainabba** [ota-meshi/jsonc-eslint-parser#142]
As I'm checking to see that I've properly implemented your plugin, I created an error in a file and tested against that one. The initial error was only a trailing comma `{ "foo", "bar", }`, but that wasn't being detected so I added an x like `{ "foo", "bar",x }`. With that error, following is my test command (eslint config done in local package.json) and the findings. 

`22:8  error  Parsing error: Expected token ':'` line 22 is the real line number where I created the error.

By default, should a trailing comma be identified as an error?

Why does that report mention a colon, but not the trialing comma or unexpected string?

Thanks for your time and efforts!

```
"eslintConfig": {
    "extends": [
      "plugin:jsonc/recommended-with-jsonc"
    ],
    "overrides": [
      {
        "files": [
          "*.json",
          "*.json5",
          "*.jsonc"
        ],
        "parser": "jsonc-eslint-parser"
      }
    ]
  }
}
```

**Comment by rainabba** [comment]
Looks like it was just my setup and I'm good now. Sorry for the noise :)
